auth0/resource_auth0_global_client.go

```diff
@@ -0,0 +1,43 @@
+package auth0
+
+import (
+	"errors"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"gopkg.in/auth0.v3"
+	"gopkg.in/auth0.v3/management"
+	"net/url"
```

Comment:
A bit of a nitpick, but can you format these similarly to the rest of the provider? `goimports` usually does the right thing.

Reply:
all taken care fo